secant/Util/WalletStorage.swift

```diff
@@ -1,18 +1,26 @@
 //
-//  RecoveryPhraseStorage.swift
+//  WalletStorage.swift
 //  secant-testnet
 //
 //  Created by Lukáš Korba on 03/10/2022.
 //
 
 import Foundation
 import MnemonicSwift
+import Security
+
+enum WalletStorageError: Error {
```

Comment:
Do you think this is better suited to live inside the namespace of WalletStorage?

Reply:
fixed